ext/fs/30_fs.js

```diff
@@ -211,31 +211,16 @@ async function rename(oldpath, newpath) {
 // 3. u64
 //  offset += 2
 //  high u32 | low u32
-//
-// 4. ?u64 converts a zero u64 value to JS null on Windows.
 function createByteStruct(types) {
   // types can be "date", "bool" or "u64".
-  // `?` prefix means optional on windows.
   let offset = 0;
-  let str =
-    'const unix = Deno.build.os === "darwin" || Deno.build.os === "linux"; return {';
+  let str = "return {";
```

Comment:
I wonder if this whole eval thingamajig is even necessary now?

Reply:
No idea, @littledivy thoughts?